src/utils/safeRename.js

```diff
@@ -0,0 +1,23 @@
+import { copyFile, rename, stat, unlink } from 'fs/promises';
```

Comment:
`fs/promises` isn't available in Node 14, I think? Hmm, but it _built_ on CI, somehow…

Any particular reason this file is in JS and not TS?

Reply:
Actually it is on node 14, but not node 12, which electron still uses.

Reply:
Yes, because I wasn't able to run TS files through jest before. But I see you did it while `childProcess.ts`, so rewrote and they're fine. We're just getting 4 warnings now about `ts-jest` instead of two.